SPHINXsys/src/shared/materials/inelastic_solid.cpp

```diff
@@ -6,17 +6,6 @@
 #include "inelastic_solid.h"
 
 namespace SPH {
-	//=================================================================================================//
-	Matd PlasticSolid::DeviatoricPK(const Matd& deviatoric_be)
-	{
-		return G0_ * deviatoric_be;
-	}
-	//=================================================================================================//
-	Real  PlasticSolid::VolumetricPK(const Matd& deformation_gradient)
-	{
-		Real J = SimTK::det(deformation_gradient);
-		return  0.5 * K0_ * J * (J - 1);
-	}
 	//=================================================================================================//
 	void HardeningPlasticSolid::initializePlasticParameters()
```

Comment:
suggest to write unit test on this function

Reply:
Yes.